pom.xml

```diff
@@ -2525,7 +2525,7 @@
               <fork>true</fork>
               <failOnError>false</failOnError>
               <meminitial>1024m</meminitial>
-              <maxmem>4048m</maxmem>
+              <maxmem>8192</maxmem>
```

Comment:
I assume the `m` is understood? It sort of makes more sense to keep it so we are explicit. Other items like `meminitial` still use the `m`.

Reply:
Yes, `m` is not required, but I added it back to keep them consistent.